chapters/annotations.tex

```diff
@@ -8,6 +8,26 @@ \chapter{Annotations}\label{annotations}
 (For replaceable class declarations with a \lstinline[language=grammar]!constraining-clause! also refer to \cref{constraining-clause-annotations}.)
 The specification in this document defines the semantic meaning if a tool implements any of these annotations.
 
+\section{Notation for Annotation Definitions}\label{notation-for-annotation-definitions}
+
+Annotations are defined using the syntactic forms of Modelica component declarations and record definitions, with the following special semantics.
+If the annotation is described by a component declaration, the annotation is used in the form of a value modifier for the same name.
+If the annotation is described by a \lstinline!record! \emph{class} the annotation is used in the form of a modifier for a \lstinline!record! \emph{component} with the same name.
+
+A declaration equation for a component or record member specifies a default to be used when no corresponding annotation is given.
+An array record member without declaration equation and with size specified by colon (\lstinline!:!) defaults to being empty.
+Default behavior can also be specified in the text as an alternative to a declaration equation, or implicitly by saying "if specified".
+If the description states the annotation only has an effect for specific values it implies that the default is a value that has no effect.
```

Comment:
Missing word:
```suggestion
If the description states that the annotation only has an effect for specific values it implies that the default is a value that has no effect.
```

Reply:
Further, is it really necessary to require that the default corresponds to any value at all?
```suggestion
If the description states that the annotation only has effect one or more specific values, the default behavior (none of these effects) does not necessarily correspond to any value.
```

Reply:
> Further, is it really necessary to require that the default corresponds to any value at all?

Perhaps not necessary - but clear as it is currently it is only used for Booleans, so the default corresponds to the other one.